changelog.json

```
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.13-beta1": [
+      "[New] Add support for notifications of pull request reviews - #14175",
+      "[Fixed] Pull requests adhere to temporal laws again - #14238",
+      "[Fixed] Fix repository group header overflow when text is too long - #14233. Thanks @tsvetilian-ty!",
+      "[Fixed] Clone dialog \"Choose\" button uses an open dialog for directory selection on macOS - #12812",
+      "[Improved] Add link to beta download page in about dialog - #14237"
```

```suggestion
```
Technically this doesn’t show in the betas but I don’t recall how release notes do or don’t build on each other 

Also this should really be a Added vs Improve

```suggestion
      "[Added] Link to beta download page in about dialog - #14237"
```

---

You're right it shouldn't be in these release notes at all, but I think `Improved` is the right category according to https://github.com/desktop/desktop/blob/241c4a2cfda89dc60d2ee7a094657ad349de7274/docs/process/writing-release-notes.md/#L129-L134

I'll remove it, thanks!! ❤️